pkg/volume/util/operationexecutor/operation_executor_test.go

```diff
@@ -224,3 +354,49 @@ func getTestPodWithGCEPD(podName, pdName string) *v1.Pod {
 		},
 	}
 }
+
+func isOperationRunSerially(ch, quit chan interface{}) bool {
+	defer close(quit)
+	numOperationsStarted := 0
+loop:
+	for {
+		select {
+		case <-ch:
+			numOperationsStarted++
+			if numOperationsStarted > 1 {
+				return false
+			}
+		case <-time.After(5 * time.Second):
```

Comment:
Ah, I misread that. 5 seconds is fine. Problem is depending on how busy the CPU is, this test could artificially pass, but oh well.